packages/kbn-eslint-plugin-imports/src/helpers/running_in_editor.ts

```diff
@@ -6,4 +6,4 @@
  * Side Public License, v 1.
  */
 
-export { getPluginSearchPaths } from './plugin_search_paths';
+export const RUNNING_IN_EDITOR = !!process.env.VSCODE_CWD;
```

Comment:
What are the needs we have to know if we are running inside VSCODE? Doesn't it simply run the same tools available for the rest of the repo?  I feel like thats what IntelliJ does, but we need to check if we have similar needs for IntelliJ then.

Reply:
Yeah, the goal is to do far less caching when running in an editor since we're highly unlikely to be checking everything and much more likely to invalidate the cache in a way we can't detect. This makes the ESLint rules more friendly than the current ones, but it's not perfect. If you can come up with a similar environment identifier for IntelliJ/WebStorm I'm happy to add it to this check!

Reply:
I was only able to identify 2 variables created by intellij:

```
"XPC_SERVICE_NAME":"application.com.jetbrains.intellij.744545809.867143471"
"__CFBundleIdentifier":"com.jetbrains.intellij"
```

Maybe we can check for `__CFBundleIdentifier` and if it contains `com.jetbrains` so it covers intellij and webstorm?

Reply:
Hmm... pretty sure that's an OSX specific thing, but we could add it. Did you try printing the `process.env` in the Eslint worker and look at the log output?

Reply:
That was exactly what I did and I've only discovered those ones related to jetbrains :cold_face: